packages/charts/examples/components/demo/chart-types/gauge/donut-with-threshold-markers/donut-gauge-with-threshold-markers-example.component.html

```diff
@@ -1,12 +1,19 @@
-<div class="d-flex flex-column align-items-center">
+<div class="d-flex align-items-center justify-content-center">
+    <div class="d-flex flex-column align-items-left mr-5">
+        <nui-form-field class="d-block mb-5" caption="Value"
```

Comment:
We may want to have `i18n-caption` here (same for other examples)

Reply:
If it's okay with you, I'll update this in my next PR. Thanks for noticing.